scenarios/bring-your-own-data/bringYourOwnData.md

```diff
@@ -0,0 +1,116 @@
+# Use Azure OpenAI On Your Data With openai-node Library
+
+## Objective
+
+The main objective of this tutorial is to help users understand the process of chatting on top of your own data. By the end of this tutorial, you should be able to:
+
+ - Connect to Azure using AzureOpenAI client
+ - Get chat response with your own data
+
+## Time
+
+You should expect to spend 5-10 minutes running this sample. 
+
+## Before you begin
+
+### Set Environment Variables
+
+To start with let us create a config `.env` file with your project details. This file can be used in this sample or other samples to connect to your workspace. A sample `.env` file is provided below with the variables that you need.
+
+```js
+OPENAI_API_VERSION="Insert the targeted API version here"
+AZURE_OPENAI_ENDPOINT="Insert your endpoint here"
+AZURE_OPENAI_API_KEY="Insert your API Key here"
+```
+
+### Connect To Your Data Sources
+We need to connect to a data source to upload our data. We will set up your own designated data sources using Azure AI Search. 
+
+```js
+const dataSources = {
+    data_sources: [
+        {
+            type: "azure_search",
+            parameters: {
+                endpoint: azureSearchEndpoint,
+                index_name: azureSearchIndexName,
+                authentication: {
+                    type: "api_key",
+                    key: azureSearchAdminKey,
+                }
+            }
+        }
+    ]
+}
+```
+
+### Using Azure OpenAI client
+We will access Azure Open AI service through `openai-node` library. 
```

Comment:
This part needs to discuss more that we need to import both identity and openai and refer to https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/README.md.